packages/dd-trace/src/plugins/util/web.js

```diff
@@ -459,6 +481,9 @@ function addResponseTags (context) {
   span.addTags({
     [HTTP_STATUS_CODE]: res.statusCode
   })
+  inferredProxySpan?.addTags({
+    [HTTP_STATUS_CODE]: res.statusCode
+  })
 
   web.addStatusError(req, res.statusCode)
```

Comment:
TODO: Figure out a way to attach the error status to an inferred span, following the web logic.

Reply:
Added a function to mirror whatever the error status of the web server span to the last commit.